From ba70d2b45b370241963dabe956b4d50a2bc4f562 Mon Sep 17 00:00:00 2001
From: Markus Elfring <elfring@users.sourceforge.net>
Date: Mon, 17 Apr 2017 11:20:41 +0200
Subject: [PATCH 1709/2241] net: mvpp2: Adjust three error messages

Use the word "failed" in the string for three function calls.

Patch cherry-picked from mainline 4.12_rc5

Change-Id: Ia9d373d770b22242a9215c7d27a7984171a0c73f
Signed-off-by: Markus Elfring <elfring@users.sourceforge.net>
Signed-off-by: David S. Miller <davem@davemloft.net>
Reviewed-on: http://vgitil04.il.marvell.com:8080/40451
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 drivers/net/ethernet/marvell/mvpp2.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/net/ethernet/marvell/mvpp2.c b/drivers/net/ethernet/marvell/mvpp2.c
index 04f3cdf..d88763d 100644
--- a/drivers/net/ethernet/marvell/mvpp2.c
+++ b/drivers/net/ethernet/marvell/mvpp2.c
@@ -6113,7 +6113,7 @@ out_start:
 	mvpp2_ingress_enable(port);
 	return 0;
 log_error:
-	netdev_err(dev, "fail to change MAC address\n");
+	netdev_err(dev, "failed to change MAC address\n");
 	return err;
 }
 
@@ -6161,7 +6161,7 @@ out_start:
 
 	return 0;
 log_error:
-	netdev_err(dev, "fail to change MTU\n");
+	netdev_err(dev, "failed to change MTU\n");
 	return err;
 }
 
@@ -6350,7 +6350,7 @@ static int mvpp2_ethtool_set_ringparam(struct net_device *dev,
 err_clean_rxqs:
 	mvpp2_cleanup_rxqs(port);
 err_out:
-	netdev_err(dev, "fail to change ring parameters");
+	netdev_err(dev, "failed to change ring parameters");
 	return err;
 }
 
-- 
2.7.4

